sematic/plugins/metrics_storage/sql/sql_metrics_storage.py

```diff
@@ -0,0 +1,239 @@
+# Standard Library
+import logging
+from typing import Dict, List, Sequence, Tuple, Type
+
+# Third-party
+from sqlalchemy import func
+from sqlalchemy.exc import NoResultFound
+
+# Sematic
+from sematic.abstract_plugin import (
+    SEMATIC_PLUGIN_AUTHOR,
+    AbstractPlugin,
+    AbstractPluginSettingsVar,
+    PluginVersion,
+)
+from sematic.db.db import db
+from sematic.metrics.metric_point import MetricPoint, MetricsLabels, MetricType
+from sematic.plugins.abstract_metrics_storage import (
+    AbstractMetricsStorage,
+    GroupBy,
+    MetricSeries,
+    MetricsFilter,
+    NoMetricError,
+)
+from sematic.plugins.metrics_storage.sql.models.metric_label import MetricLabel
+from sematic.plugins.metrics_storage.sql.models.metric_value import MetricValue
+from sematic.utils.hashing import get_str_sha1_digest
+
+logger = logging.getLogger(__name__)
+
+
+_PLUGIN_VERSION = (0, 1, 0)
+
+
+class SQLMetricsStorageSettingsVar(AbstractPluginSettingsVar):
+    pass
```

Comment:
Should eventually allow for customizing the DB to which metrics are written. Currently write to the Sematic DB.